submissions/Chimoney-Python/chimoney/Info.py

```diff
@@ -15,7 +15,7 @@ def airtime_countries(self) -> dict:
         """
         return self._handle_request("GET", "/v0.2/info/airtime-countries")
 
-    def assets(self, country_code: str = None) -> dict:
```

Comment:
Hello, with `None` and `str` you can use `Optional` like this:
```py
x: Optional[str] = "something" if some_condition() else None
```

More about [here.](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#useful-built-in-types)

Reply:
I am not sure this is necessary in the mean time but I'll look into it
@thelimeskies what do you think?